src/hooks/useGame.hook.js

```diff
@@ -21,17 +21,18 @@ export default function useGame(marksToWin = 3) {
       return
     }
 
-    if (marksInBoard === NUMBER_OF_POSITIONS) {
-      alert('DRAW GAME!')
-      handleOnReset()
-      return
-    }
-
     const theWinner = getWinner(board)
 
     if (theWinner) {
       setWinner(theWinner)
       setNextPlayer(NO_PLAYER)
+      return
+    }
+
+    if (marksInBoard === NUMBER_OF_POSITIONS) {
```

Comment:
In this method it is necessary to add a validation where we indicate if the variable theWinner is different from null

Reply:
In this case the validation is not needed because at this point if the winner is not returned by the previous block and the marks in the board reach the number of positions, the game is over.